### PR TITLE
Footer copyright and payment

### LINKED ISSUE #23: Add copyright to footer
# Current text:

Made with love and brought to you by Kloud51, a Savand Bros brand <3 © 2015-2017 Savand Bros. Kloud51. All Rights Reserved.

# Plus payment methods:

image

### PR DESCRIPTION
**Pull request by ArsalanSavand**
Resolves #23

### CODE REVIEW

File: app/base.php [hunk 1]
```diff
@@ -199,6 +199,49 @@
...
+      <div class="container">
+        <div class="row">
+          <!-- Copyright -->
+          <div class="col-lg-3">
+            <a href="/affiliate-army-program">
+              <img src="assets/img/logo-white.png">
+            </a>
+          </div>
+          <!-- Goal -->
+          <div class="col-lg-9 goal-content">
```
Comment (AmirSavand) [inline app/base.php]: Remove the class, not necessary.

File: app/base.php [hunk 2]
```diff
@@ -199,6 +199,49 @@
...
+            <p>
+              Our goal is to provide out of this world web hosting hosting and Linux SSD VPS services,
+              become a loved brand and have a little fun on the side.
+              We offer Premium web hosting services at low prices to our customers without compromising reliability and great customer service.
```
Comment (AmirSavand) [inline app/base.php]: Too long, break the sentence.

File: app/assets/css/app.css [hunk 3]
```diff
@@ -307,3 +308,34 @@ p {
...
+    text-shadow: -1px -1px #111;
+}
+
+/* Footer goal */
+
+.footer .goal {
+    background: #111;
+}
+
+.footer .goal .goal-content {
```
Comment (AmirSavand) [inline app/assets/css/app.css]: Use `p` selector, class is not necessary.

**Review by AmirSavand — CHANGES_REQUESTED** [review]

File: app/base.php (hunk 2 above)
Comment (ArsalanSavand) [reply app/base.php]: Done.

File: app/base.php (hunk 1 above)
Comment (ArsalanSavand) [reply app/base.php]: Done.

File: app/assets/css/app.css (hunk 3 above)
Comment (ArsalanSavand) [reply app/assets/css/app.css]: Done.

**Review by AmirSavand — APPROVED** [review]

**Pull request merged**